app/controllers/Contributions.java

```diff
@@ -2246,19 +2246,34 @@ public static Contribution createContribution(Contribution newContrib,
         Logger.info("Using Etherpad server at: " + etherpadServerUrl);
         Logger.debug("Using Etherpad API Key: " + etherpadApiKey);
         
-        Campaign ca = Campaign.readByResourceSpaceId(containerResourceSpace.getResourceSpaceId());        
-        if (ca != null){
-        	List<Config> campaignConfigs = Config.findByCampaignAndKey(ca.getUuid(), "appcivist.campaign.disable-etherpad");        	
+        if (containerResourceSpace.getType().equals(ResourceSpaceTypes.CAMPAIGN)) {
+            Campaign c = containerResourceSpace.getCampaign();    	        
+        	List<Config> campaignConfigs = c.getConfigs();
+        	Integer setStatus = 0;        	
         	for(Config cc: campaignConfigs){
-        		if (cc.getValue().equalsIgnoreCase("FALSE")){
-        	        if (type != null && (type.equals(ContributionTypes.PROPOSAL) || type.equals(ContributionTypes.NOTE))) {
+        		if (type != null && type.equals(ContributionTypes.PROPOSAL)) {
+        			if (cc.getKey().equals("appcivist.campaign.proposal-default-status")) {        	
```

Comment:
- [ ] Same comment as with the config for disabling etherpad support: use GlobalConfigKeys